authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import serializers
+from .models import Article
+from ..authentication.models import User
+from taggit_serializer.serializers import (TagListSerializerField,
+                                           TaggitSerializer)
+
+class CreateArticleAPIViewSerializer(TaggitSerializer,serializers.ModelSerializer):
+    tags = TagListSerializerField()
+    author = serializers.SerializerMethodField()
+
+    def get_author(self,obj):
+        user = {
+            "username":obj.author.username,
+            "email":obj.author.email
+        }
+        return user
+
+
+    class Meta:
+        model = Article
+
+        fields = ['title','description', 'body', 'author', 
+                    'created_at', 'updated_at', 'tags', 'slug', 'published', 'image']
```

Comment:
I would also advise that you implement the tagList change here to as it affects the article posting documentation we are currently using

`{
  "article": {
    "title": "How to train your dragon",
    "description": "Ever wonder how?",
    "body": "You have to believe",
    "tagList": ["reactjs", "angularjs", "dragons"]
  }
}`